x-pack/plugins/ml/public/timeseriesexplorer/timeseriesexplorer.html

```diff
@@ -54,25 +73,44 @@
 
   <div class="no-results-container" ng-show="jobs.length > 0 && loading === false && hasResults === false">
     <div class="no-results">
-      <div><i class="fa fa-info-circle" ></i>No results found</div>
-      <div>Try widening the time selection or moving further back in time</div>
+      <div
+        i18n-id="xpack.ml.timeSeriesExplorer.noResultsFoundLabel"
+        i18n-default-message="{icon} No single metric jobs found"
```

Comment:
Text should be "Try widening the time selection or moving further back in time", copy/paste typo